frlzjosh/project_management_system
# PMS-4 | Installed yarn

## Ready/Hold
`Is this ready to merge?` 

## Description
`What does this branch do?`
- installed yarn

## Steps for Testing
`How do we check if this works?`
- try doing 'npm install yarn' and then 'yarn run dev' to see that t works'
